src/components/themed/SimpleTextInput.tsx

```diff
@@ -179,52 +184,61 @@ export const SimpleTextInput = React.forwardRef<SimpleTextInputRef, SimpleTextIn
-            // Callbacks:
-            onBlur={handleBlur}
-            onChangeText={onChangeText}
-            onFocus={handleFocus}
-            onSubmitEditing={handleSubmitEditing}
-            maxLength={maxLength}
-            // Other Props:
-            autoCapitalize={autoCapitalize}
-            autoCorrect={autoCorrect}
-            blurOnSubmit={blurOnSubmit}
-            inputAccessoryViewID={inputAccessoryViewID}
-            secureTextEntry={secureTextEntry}
-          />
-        </InnerContainer>
-
-        <TouchContainer accessible onPress={handleClearPress} testID={`${testID}.clearIcon`}>
-          <SideContainer size={rightIconSize}>
-            <CloseIconAnimated color={iconColor} size={rightIconSize} />
-          </SideContainer>
-        </TouchContainer>
-      </Container>
-    </EdgeTouchableWithoutFeedback>
+    <ContainerView>
+      <EdgeTouchableWithoutFeedback accessible={false} testID={testID} onPress={() => focus()}>
+        <InputContainerView disableAnimation={disableAnimation} focusAnimation={focusAnimation} scale={scale} spaceProps={spaceProps}>
+          <SideContainer size={leftIconSize}>{Icon == null ? null : <Icon color={iconColor} size={leftIconSize} />}</SideContainer>
+
+          <InnerContainer>
+            <InputField
+              accessible
+              ref={inputRef}
+              keyboardType={props.keyboardType}
+              returnKeyType={props.returnKeyType}
+              accessibilityState={{ disabled }}
+              autoFocus={autoFocus}
+              disableAnimation={disableAnimation}
+              focusAnimation={focusAnimation}
+              placeholder={placeholder}
+              placeholderTextColor={placeholderTextColor}
+              selectionColor={theme.textInputTextColor}
+              testID={`${testID}.textInput`}
+              textAlignVertical="top"
+              value={value}
+              // Callbacks:
+              onBlur={handleBlur}
+              onChangeText={onChangeText}
+              onFocus={handleFocus}
+              onSubmitEditing={handleSubmitEditing}
+              maxLength={maxLength}
+              // Other Props:
+              autoCapitalize={autoCapitalize}
+              autoCorrect={autoCorrect}
+              blurOnSubmit={blurOnSubmit}
+              inputAccessoryViewID={inputAccessoryViewID}
+              secureTextEntry={secureTextEntry}
+            />
+          </InnerContainer>
+
+          <TouchContainer accessible onPress={handleClearPress} testID={`${testID}.clearIcon`}>
+            <SideContainer size={rightIconSize}>
+              <CloseIconAnimated color={iconColor} size={rightIconSize} />
+            </SideContainer>
+          </TouchContainer>
+        </InputContainerView>
+      </EdgeTouchableWithoutFeedback>
+      {isFocused ? (
+        <ButtonUi4 onPress={handleDonePress} type="tertiary" label={lstrings.string_done_cap} testID={`${testID}.doneButton`} marginRem={[0.5, 0, 0.5, 0.5]} />
```

Comment:
`marginRem={0.5}`

Out of the scope of this task but just noting for later - should make the defaults 0.5 as a default, and always override when using within a `ButtonsView`

Reply:
The margin is set to 0.5 on each side except for the right side

Reply:
Preferably we don't tile things in a way where one component handles 100% of the spacing on any one side. By splitting up the margins, we're able to minimize the amount of hard-coded margins, and allow things to be reused easier by others without knowledge of the actual implementation of the styling since they all follow the same expectations.